src/SimpleSimulator.cpp

```diff
@@ -284,6 +309,17 @@ int main(int argc, char **argv) {
     // Set XRootD block size
     SimpleSimulator::xrd_block_size = vm["xrd-blocksize"].as<double>();
 
+    // Choice of cache locality scope
+    std::string scope_caches = vm["cache-scope"].as<std::string>();
+    bool rec_netzone_caches;
+    if (scope_caches.find("network") == std::string::npos) {
+        SimpleSimulator::local_cache_scope = true;
+    } else {
+        if (scope_caches.find("rec") != std::string::npos) {
+            rec_netzone_caches = true;
+        }
+    }
+
```

Comment:
Can be restricted to 3 choices, which can be introduced above.

Reply:
Done